test/writer.test.js

```diff
@@ -43,6 +43,17 @@ describe('writer', function () {
       });
     });
 
+    it('keeps only the date if no version is specified', function () {
+      var options = { major: true };
+
+      return Writer.markdown(false, [], options)
+      .then(function (changelog) {
+        var heading = changelog.split('\n')[0];
+
+        Expect(heading).to.equal('## ' + new Date().toJSON().slice(0,10));
```

Comment:
Nit pick: can you add a space after the comma in the `.slice`? There should've been a lint rule for that. I'll look into why that didn't err out.

Reply:
Actually I copied this from [writer.js](https://github.com/lob/generate-changelog/blob/39e7fd93e3fbff641c3582b6192d3b32c4c68550/lib/writer.js#L34) but I see that you updated it with the new lint rule :)